docs/index.md

```diff
@@ -2,12 +2,12 @@
 # todo: internal
 ---
 
-# Prometheus 2.0
+# Prometheus 2.7
```

Comment:
Let's remove the version here, as we're not updating it.

Reply:
Thanks @brian-brazil. I have removed the version as your reviewing :)